.drone.env

```diff
@@ -1,5 +1,5 @@
 # The test runner source for API tests
-CORE_COMMITID=f16ee06034a041433e53fb3d86145d29155ea3d9
+CORE_COMMITID=c9acf31ed161b66264ac5022b0d3fe148f349222
```

Comment:
Do we need this?
Didn't you remove the tests added here from core and also removed from expected to failure?